Examples/python/class_with_slots/example.i

```diff
@@ -0,0 +1,28 @@
+/* File : example.i */
+%module example
+
+%{
+    #include "example.h"
+%}
+
+
+%extend Shape{
```

Comment:
the only difference between the two tests is the addition of the slots. 

Most of what slots do for me is they help me avoid typos once the API get mostly stable.